### PR TITLE
Update voltageLevel mapping

### PR DESCRIPTION
**Pull request by ayolab**

### CODE REVIEW

File: src/main/java/org/gridsuite/network/map/NetworkMapService.java [hunk 1]
```diff
@@ -51,18 +53,57 @@ private static VoltageLevelMapData toMapData(VoltageLevel voltageLevel, boolean
...
+                warning.set(true);
+            }
+        });
+        builder.isPartiallyCopied(warning.get());
+        if (!warning.get()) {
+            builder.busbarCount(busbarCount.get());
+            builder.sectionCount(sectionCount.get());
+            SwitchKind[] switchKinds = new SwitchKind[sectionCount.get() - 1];
+            Arrays.fill(switchKinds, SwitchKind.DISCONNECTOR);
+            builder.switchKinds(Arrays.asList(switchKinds));
```
Comment (FranckLecuyer) [inline src/main/java/org/gridsuite/network/map/NetworkMapService.java:99]: builder.switchKinds(new ArrayList<SwitchKind>(Collections.nCopies(sectionCount.get() - 1, SwitchKind.DISCONNECTOR));    in one single line ?

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]

**Review by FranckLecuyer — APPROVED** [review]

File: src/main/java/org/gridsuite/network/map/NetworkMapService.java (hunk 1 above)
Comment (ayolab) [reply src/main/java/org/gridsuite/network/map/NetworkMapService.java]: Done

File: src/main/java/org/gridsuite/network/map/model/VoltageLevelMapData.java [hunk 2]
```diff
@@ -32,10 +33,23 @@ public class VoltageLevelMapData {
 
     private double nominalVoltage;
 
+    private double lowVoltageLimit;
```
Comment (FranckLecuyer) [inline src/main/java/org/gridsuite/network/map/model/VoltageLevelMapData.java:36]: @JsonInclude(JsonInclude.Include.NON_NULL)
private Double lowVoltageLimit;

same for other double fields

File: src/main/java/org/gridsuite/network/map/model/VoltageLevelMapData.java [hunk 3]
```diff
@@ -32,10 +33,23 @@ public class VoltageLevelMapData {
...
+
+    private double highVoltageLimit;
+
+    private Double ipMin;
+
+    private Double ipMax;
+
+    private int busbarCount;
```
Comment (FranckLecuyer) [inline src/main/java/org/gridsuite/network/map/model/VoltageLevelMapData.java:44]: @JsonInclude(JsonInclude.Include.NON_NULL)
private Integer busbarCount;

same for other int fields

File: src/main/java/org/gridsuite/network/map/model/VoltageLevelMapData.java [hunk 4]
```diff
@@ -32,10 +33,23 @@ public class VoltageLevelMapData {
...
+
+    private int sectionCount;
+
+    private List<SwitchKind> switchKinds;
```
Comment (FranckLecuyer) [inline src/main/java/org/gridsuite/network/map/model/VoltageLevelMapData.java:48]: Add @JsonInclude(JsonInclude.Include.NON_NULL)

File: src/main/java/org/gridsuite/network/map/model/VoltageLevelMapData.java [hunk 5]
```diff
@@ -32,10 +33,23 @@ public class VoltageLevelMapData {
 
     private double nominalVoltage;
 
+    private double lowVoltageLimit;
+
+    private double highVoltageLimit;
+
+    private Double ipMin;
```
Comment (FranckLecuyer) [inline src/main/java/org/gridsuite/network/map/model/VoltageLevelMapData.java:40]: Add @JsonInclude(JsonInclude.Include.NON_NULL)

File: src/main/java/org/gridsuite/network/map/model/VoltageLevelMapData.java [hunk 6]
```diff
@@ -32,10 +33,23 @@ public class VoltageLevelMapData {
...
+
+    private Double ipMax;
```
Comment (FranckLecuyer) [inline src/main/java/org/gridsuite/network/map/model/VoltageLevelMapData.java:42]: Add @JsonInclude(JsonInclude.Include.NON_NULL)

File: src/main/java/org/gridsuite/network/map/model/VoltageLevelMapData.java [hunk 7]
```diff
@@ -32,10 +33,23 @@ public class VoltageLevelMapData {
...
+    private int sectionCount;
+
+    private List<SwitchKind> switchKinds;
+
     @JsonInclude(JsonInclude.Include.NON_NULL)
     private TopologyKind topologyKind;
 
-    @JsonInclude(JsonInclude.Include.NON_NULL)
-    private List<BusbarSectionMapData> busbarSections;
+    private Boolean isPartiallyCopied;
```
Comment (FranckLecuyer) [inline src/main/java/org/gridsuite/network/map/model/VoltageLevelMapData.java:53]: Add @JsonInclude(JsonInclude.Include.NON_NULL)

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]

**Review by FranckLecuyer — APPROVED** [review]

**Review by YenguiSeddik — APPROVED** [review]

**Pull request merged**